### PR TITLE
Fix button styles

### LINKED ISSUE #42: Fix button styling on frontend
The Course List buttons currently look like this in the editor:

Screen Shot 2022-10-17 at 9 40 46 AM

But they look like this on the frontend:

Screen Shot 2022-10-17 at 11 10 36 AM

We should update all Course List buttons so that they look similar to the [design](https://www.figma.com/file/nf4oV51c1JBxJKXshMe7cd/New-Sensei-Theme?node-id=20020078%3A3018):

Screen Shot 2022-10-17 at 9 43 24 AM

#### Scenario: Course List buttons - Editor
**Given** The Sensei theme is activated
**When** I add the Course List block to a page
**Then** the _Start Course_ buttons should look similar to the design in the editor.

#### Scenario: Course List buttons - Frontend
**Given** The Sensei theme, Sensei LMS, Sensei Pro and WooCommerce are activated
**And** I've created at least 4 courses, one of which is paid
**And** I've started one free course and completed another free course
**When** I view a page containing the Course List block
**Then** the _Start Course_, _Buy_, _Continue_ and _Visit Results_ buttons should look similar to the design on the frontend.

### PR DESCRIPTION
**Pull request by donnapep**
Fix button styles to match the [design](https://www.figma.com/file/nf4oV51c1JBxJKXshMe7cd/New-Sensei-Theme?node-id=20020042%3A329). In particular:

- Button height is 56px.
- Left & right padding is 32px.
- Font size is 21px.
- Letter spacing is 0.05em.
- Fix the button active, hover and focus states.

Note that the button background and text color don't match the design, but this will be addressed in a separate PR.

For the Course List block, remove the link underline from all buttons for all button states.

## Testing Instructions
- Switch to [this branch of](https://github.com/Automattic/sensei/pull/6010) Sensei.
- Add some buttons to a page.
- View the page in the editor and on the frontend and ensure the button styles are correct.
- Add the Course List block to a page.
- View the page in the editor and on the frontend and ensure the button styles are correct and the button text is not underlined for any of the button states.

### Default
Screen Shot 2022-10-26 at 10 05 49 AM

### Hover
Screen Shot 2022-10-26 at 10 06 27 AM

### Focus
Screen Shot 2022-10-26 at 10 05 25 AM

### Active
Screen Shot 2022-10-26 at 10 07 12 AM

### Course List
Screen Shot 2022-10-26 at 11 28 47 AM

### CODE REVIEW

File: theme.json [hunk 1]
```diff
@@ -48,7 +48,19 @@
 			"replyColumnGap": "2.349rem",
 			"commentGap" : "5rem",
 			"commentGapSmall" : "10px",
-			"commentGapMedium" : "1.25rem"
+			"commentGapMedium" : "1.25rem",
+			"button": {
+				":active": {
+					"border": {
+						"color": "#FFF7BC"
```
Comment (gabrielcaires) [inline theme.json]: I think we should move to the color pallet to enable the users to customize it.
I didn't find a way to customize it.

**Review by gabrielcaires — CHANGES_REQUESTED** [review]
Thanks for fixing it.
I just left a comment related to the colors

File: theme.json (hunk 1 above)
Comment (donnapep) [reply theme.json]: Even if I add it to the color palette, I don't think WordPress offers a way to customize the border color of different button states (`active` and `hover` in this case). Did you find a place to do this?

Comment (gabrielcaires) [reply theme.json]: @donnapep I didn't find it, but I think if we add it as a normal pallet with a descriptive name will be enough to enable the users to customize it.  

The current state if the user changes the green tones to red they will still have elements in green.

What do you think?


Screen Shot 2022-10-27 at 10 02 34

Comment (donnapep) [reply theme.json]: I think you're referring to the fill button style, correct? If so, there was an easy fix for that in https://github.com/Automattic/sensei-theme/pull/52/commits/cbc10bd4b6d8963f44a4f530c04737d76a26950e. Now when you change the background color, there is no more border. 🙌🏻

Comment (gabrielcaires) [reply theme.json]: > I think you're referring to the fill button style, correct? If so, there was an easy fix for that in [cbc10bd](https://github.com/Automattic/sensei-theme/commit/cbc10bd4b6d8963f44a4f530c04737d76a26950e). Now when you change the background color, there is no more border. 🙌🏻

Hmn, maybe not.

I am talking more about it:
[Screen Recording on 2022-10-27 at 14_32_34.webm](https://user-images.githubusercontent.com/38718/198359305-be9b4801-e4a9-4cd5-9bc9-7172e8df3de8.webm)

The border color is set on the theme.json Custom Field which doesn't enable the users to customize.
On the video case, I  update all green tones to red tones but the border is still green and I don't have a way to customize it. 

Screen Shot 2022-10-27 at 14 36 10

Comment (donnapep) [reply theme.json]: OK, all set in https://github.com/Automattic/sensei-theme/pull/52/commits/62fc8be599540f625b7c19250c7bfbf76308ffff. (Sorry, I did a force push. 😐 )

**Review by gabrielcaires — APPROVED** [review]

**Pull request merged**